.github/workflows/build-docs.yml

```diff
@@ -105,6 +105,7 @@ jobs:
         uses: tardis-sn/tardis-actions/setup-env@main
         with:
           os-label: linux-64
+          lock-file-url-prefix: "https://raw.githubusercontent.com/tardis-sn/tardis/master"
```

Comment:
is there a way to use the local lock file? I don't think we should download. It should pass with the one that is in the repo not the one on master